### PR TITLE
Use the correct terminology

### PR DESCRIPTION
**Pull request by Sakrecoer**
"Secret Recovery Phrase" instead of "Secret Phrase"

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/current/wallet/Desktop-Wallet-Guide.md [hunk 1]
```diff
@@ -3,10 +3,6 @@ sidebar_position: 30
 title: Desktop Wallet Guide
 ---
 
-:::info
-Cette page n'a pas encore été traduite. Vous pouvez le faire en suivant le lien en pied de page.
-:::
-
```
Comment (nop33) [inline i18n/fr/docusaurus-plugin-content-docs/current/wallet/Desktop-Wallet-Guide.md]: Why did you remove this?

**Review by nop33 — COMMENTED** [review]
👏

File: i18n/fr/docusaurus-plugin-content-docs/current/wallet/Desktop-Wallet-Guide.md (hunk 1 above)
Comment (Sakrecoer) [reply i18n/fr/docusaurus-plugin-content-docs/current/wallet/Desktop-Wallet-Guide.md]: I did it inadvertently. Good catch, thanks :)

Comment (Sakrecoer) [reply i18n/fr/docusaurus-plugin-content-docs/current/wallet/Desktop-Wallet-Guide.md]: fixed!

**Review by nop33 — APPROVED** [review]
